`make fmt` doesn't work with a newer version of rustfmt

If we are going to require rust nightly to build ekiden,  `make fmt` should work with a nightly version of `rustfmt` as well.

## Details

```
arnhem :: oasislabs/repos/ekiden ‹yawning/feature/runtime-persistent-rak› % rustfmt --version
rustfmt 1.2.0-nightly (09940a7 2019-03-27)
arnhem :: oasislabs/repos/ekiden ‹yawning/feature/runtime-persistent-rak› % make fmt
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
Error: rustfmt version (1.2.0) doesn't match the required version (1.1.0)
Error writing files: version mismatch
make: *** [Makefile:67: fmt] Error 1
```

## Acceptance Criteria
* What indicates the issue is resolved?

`make fmt` working.

--- a/.rustfmt.toml
+++ b/.rustfmt.toml
@@ -50,7 +50,6 @@ use_field_init_shorthand = false
 force_explicit_abi = true
 condense_wildcard_suffixes = false
 color = "Auto"
-required_version = "1.1.0"
 unstable_features = false
 disable_all_formatting = false
 skip_children = false

